README.md

```diff
@@ -1,2 +1,8 @@
 # GitHubAiAPI
 This is Example to test AI plugin for automatic code review
```

Comment:
The title 'GitHubAiAPI' could be more descriptive. Consider providing a brief explanation of what the API does.